doc/Development_Documentation/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md

```diff
@@ -87,7 +87,7 @@ Please make sure to set your preferred storage location ***before*** migration.
 - Removed Pimcore Password Encoder factory, `pimcore_admin.security.password_encoder_factory` service and `pimcore.security.factory_type` config.
 - Removed BruteforceProtection
 - Removed PreAuthenticatedAdminToken
-- 
+- [Codeception] Bumped `codeception/codeception` version to ^5.0
```

Comment:
We should also mention that Pimcore is using new directory structure for tests, which breaks the existing implementations relying on the Pimcore test classes & support helpers.

Reply:
Ok 👍